src/components/NavigationButtons.js

```diff
@@ -56,6 +61,12 @@ export default ({
           </StyledButton>
         </div>
       </ButtonContainer>
+      {showSubmitWarning && (
```

Comment:
Can we move this to underneath the autosave text, right before `<ButtonContainer>`?
Also let's replace <P> and <StyledAsterisk> with just
```
<ErrorMessage>Caution! You cannot edit your application after submitting.</ErrorMessage>
```
`ErrorMessage` is defined in Typography

Reply:
![image](https://user-images.githubusercontent.com/56183965/142506284-9b913527-42cc-4482-b29f-147de8b868d8.png)

Done! Now it looks like the above image.